packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -182,6 +186,7 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
       onClick={() => {
         setItemHovered(true);
         setMenuHidden(false);
+        onClick?.();
```

Comment:
You can do that?! (the `?.()`) I love it! No more `fn && fn()`

Reply:
Yup. Recently learned it :)